tests/operator_test.go

```diff
@@ -737,6 +738,29 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 				vmis = append(vmis, vmi)
 			}
 
+			lastVMIIndex := len(vmis) - 1
+			vmi := vmis[lastVMIIndex]
+			const nadName = "secondarynet"
+
+			Expect(libnet.CreateNAD(virtClient, vmi.GetNamespace(), nadName)).To(Succeed())
+
+			vmi.Spec.Domain.Devices.Interfaces = append(
+				vmi.Spec.Domain.Devices.Interfaces,
+				v1.Interface{
+					Name:                   "tenant-blue",
```

Comment:
nit: Could you please define `"tenant-blue"` as a constant so we can see the relation between the iface and net?